game/tests.py

```
@@ -69,8 +68,23 @@ def test_falls_back_to_python_engine_script(self):
             )
 
 
-class BoardViewTest(TestCase):
-    """The board page should load and initialise a session."""
+
+class AuthenticatedClientTestCase(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.user = User.objects.create_user(username='testuser', password='password')
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "password"

(S106)